actions/debug

```diff
@@ -64,7 +64,17 @@ def run_script(script):
                 "debug-scripts/" + script,
                 stdout=stdout, stderr=stderr, env=env
             )
-            exit_code = process.wait()
+            try:
+                exit_code = process.wait(timeout=300)
+            except subprocess.TimeoutExpired:
+                log("ERROR: still running, terminating")
```

Comment:
Will it be obvious what process this is or should we output the script name as well?

Reply:
Probably wouldn't hurt, but in the interest of time I want to get some more runs kicked off tonight